boa_engine/src/job.rs

```diff
@@ -1,12 +1,37 @@
-//! Data structures for the microtask job queue.
+//! `ECMAScript` jobs and job queues.
```

Comment:
Small doc nitpick. I think this could probably be replaced with a modified version of the sentence below. Something like: "Boa's implementation and API to create and customize ECMAScript jobs and job queues."